README.md

```diff
@@ -0,0 +1,17 @@
+Tribler
```

Comment:
Just so you are aware, this file is very unlikely to be ever seen by a user. This description belongs to appdata file and not here.

Reply:
Sure! I guess we should just delete it.